pages/integrations/sso.md.erb

```diff
@@ -47,6 +47,18 @@ For more information on the SAML user attributes Buildkite supports, see the [Cu
 
 If you'd prefer a self-service setup, you can complete the Buildkite side yourself using the Buildkite GraphQL explorer. Check out our guide to [Setting up SSO with GraphQL](/docs/integrations/sso/sso-setup-with-graphql).
 
+## How do I disable SSO?
+
+If you need to edit your SSO settings, temporarily stop logins using SSO, or want to delete your SSO provider, you'll first need to disable it.
+
+There are two ways to disable a provider:
+- using the 'disable' button in your SSO provider Settings
+- using GraphQL
```

Comment:
Do we usually add references or examples for graphql things?

Reply:
Yeah, I'll link to the graphql sso guide, it's got example code 👍🏻

Reply:
Ahhh, of course! Bueno 👌